x/group/types.go

```diff
@@ -31,10 +31,13 @@ type DecisionPolicy interface {
 	// GetVotingPeriod returns the duration after proposal submission where
 	// votes are accepted.
 	GetVotingPeriod() time.Duration
+	// GetMinExecutionPeriod returns the minimum duration after submission
+	// where we can execution a proposal.
+	GetMinExecutionPeriod() time.Duration
```

Comment:
There's a way to make this addition non-breaking, using extension interfaces. I'm not sure it's really desirable (this PR is anyways state-breaking), as I'm not sure anyone created other group policies than the 2 we have in the SDK.